### PR TITLE
feat(build): #1157 update vuln dependencies

### PR DESCRIPTION
**Pull request by rohaquinlop**
- Update fast-json-patch version

### CODE REVIEW

File: src/args/lint-with-ajv/ajv-cli/package.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
-    "ajv-cli": "*"
+    "ajv-cli": "*",
+    "fast-json-patch": "3.1.1"
```
Comment (dsalaza4) [inline src/args/lint-with-ajv/ajv-cli/package.json:4]: @dacevedo12 Is this what do we do when we want to update a vulnerable sub-dependency?

Comment (dacevedo12) [reply src/args/lint-with-ajv/ajv-cli/package.json:4]: as can be seen in the package lock diff line 293, this didn't have the intended effect.

it still installs fast-json-patch 2.2.1

you can use overrides to force npm to install the version you desire

Comment (dsalaza4) [reply src/args/lint-with-ajv/ajv-cli/package.json:4]: @rohaquinlop

Comment (dacevedo12) [reply src/args/lint-with-ajv/ajv-cli/package.json:4]: more info here https://docs.npmjs.com/cli/v10/configuring-npm/package-json#overrides

Comment (rohaquinlop) [reply src/args/lint-with-ajv/ajv-cli/package.json:6]: Thanks!!!

File: src/args/lint-with-ajv/ajv-cli/package.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "dependencies": {
     "ajv-cli": "*"
+  },
+  "overrides": {
```
Comment (dsalaza4) [inline src/args/lint-with-ajv/ajv-cli/package.json:5]: @rohaquinlop Did you test the builtin?

Comment (rohaquinlop) [reply src/args/lint-with-ajv/ajv-cli/package.json:5]: Yes

Comment (rohaquinlop) [reply src/args/lint-with-ajv/ajv-cli/package.json:5]: image

**Review by dsalaza4 — APPROVED** [review]
lgtm

**Pull request merged**